docs/partials/api.md

```diff
@@ -144,9 +144,9 @@ The `auro-menuoption` element supports the placement of the `disabled` attribute
 
 </auro-accordion>
 
-#### Menu with pre-selected menu option
+#### `selected`
```

Comment:
```suggestion
#### selected
```

The backticks in headers is an old style we tried and backed away from. Still lingers in a few places but we need to stop using it in headers.